web/modules/custom/profile_block/src/Controller/Controller.php

```diff
@@ -24,29 +22,53 @@ class Controller extends ControllerBase {
    */
   private $connection;
 
+  /**
+   * The current route match.
+   *
+   * @var \Drupal\Core\Routing\CurrentRouteMatch
+   */
+  protected $currentRouteMatch;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManager
+   */
+  protected $entityTypeManager;
+
   /**
    * The database connection to be used.
    *
    * @param \Drupal\Core\Database\Connection $connection
    *   The database connection to be used.
+   * @param \Drupal\Core\Routing\CurrentRouteMatch $currentRouteMatch
+   *   The current route match.
+   * @param \Drupal\Core\Entity\EntityTypeManager $entity_type_manager
+   *   The entity type manager.
    */
-  public function __construct(Connection $connection) {
+  public function __construct(Connection $connection, CurrentRouteMatch $currentRouteMatch, EntityTypeManager $entity_type_manager) {
     $this->connection = $connection;
+    $this->currentRouteMatch = $currentRouteMatch;
+    $this->entity_type_manager = $entity_type_manager;
```

Comment:
$this->entity_type_manager should be $this->entityTypeManager right

Reply:
fixed here: https://github.com/guptahemant/insta-clone/pull/79/commits/33d95104fb8e57192cee781f012affa9093c6567#diff-27ed9e15a36bba030870a9e341b1c19a7d0857dec15759d84cd2029f8965dd24